fundamentos/Bloco-03-Introducao-HTML-CSS/dia-4-html-semantico/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Oi Rodolfo, você pode alterar sua tag html desta forma:
```suggestion
<html lang="pt-br">
```